src/mock/ray/raylet_client/raylet_client.h

```diff
@@ -19,8 +19,8 @@ class MockPinObjectsInterface : public PinObjectsInterface {
   MOCK_METHOD(void,
               PinObjectIDs,
```

Comment:
Should we rename the interface to PinObjectID?

Reply:
Yeah, let's just leave them as they are for now.

Reply:
I feel we probably don't have to make their name exactly the same since they are different: one is pinning a single object and the rpc one is pinning multiple objects so PinObjectID and PinObjectIDs{Request|Reply} reflect that.